test/Libraries/DynamoMSOfficeTests/Properties/AssemblyInfo.cs

```diff
@@ -1,10 +1,12 @@
-﻿using System.Reflection;
+using NUnit.Framework;
+using System.Reflection;
 using System.Runtime.InteropServices;
+using System.Threading;
```

Comment:
I suggest you revert the changes to this file

Reply:
@marbelarrietaGlobant please revert all changes to this file. 
I still see some whitespace changes.